bluemira/geometry/wire.py

```diff
@@ -61,6 +61,8 @@ def __init__(
         self, boundary: List[Union[cadapi.apiWire, BluemiraWire]], label: str = ""
     ):
         boundary_classes = [self.__class__, cadapi.apiWire]
+        if isinstance(boundary, cadapi.apiWire):
+            cadapi.fix_wire(boundary)
```

Comment:
Please could you do a rough time comparison on the EU-DEMO CAD build sequence with this fixing on and off?

I'd like to get an understanding of the overhead, and whether it's worth applying to all `BluemiraWire`s rather than addressing the root cause of the particular issue in this case.

Reply:
I've run EUDEMO a couple of time with and without these changes.

The full reactor build takes around 96 seconds to finish with tf and pf designers set to read run_mode

It takes in the region of 1.5-2 seconds longer to build with these changes, which is essentially negligible compared to the total time it takes to build.
It seems like very little overhead is added and it doesn't "feel" any slower when running.

I used `time.perf_counter()` to read the time before and after all the build_x functions are run in `reactor.py`